internal/engine/executor.go

```diff
@@ -191,7 +191,7 @@ func (e *Executor) evalEntityEvent(ctx context.Context, inf *entities.EntityInfo
 			// Let's evaluate all the rules for this profile
 			err = profiles.TraverseRules(relevant, func(rule *pb.Profile_Rule) error {
 				// Get the engine evaluator for this rule type
-				evalParams, ruleEngine, actions, err := e.getEvaluator(
+				evalParams, ruleEngine, actionEngine, err := e.getEvaluator(
```

Comment:
Two small changes in this file:

1) Rename `actions` to `actionEngine` to avoid shadowing an import name
2) Move `SetActionsErr` directly under the call to run actions.